app/controllers/manageBusinesses/cease/IncomeSourceEndDateController.scala

```diff
@@ -86,7 +86,7 @@ class IncomeSourceEndDateController @Inject()(val authorisedFunctions: FrontendA
 
   def handleRequest(id: Option[IncomeSourceIdHash], isAgent: Boolean, isChange: Boolean, incomeSourceType: IncomeSourceType)
                    (implicit user: MtdItUser[_], ec: ExecutionContext): Future[Result] =
-    withSessionData(JourneyType(Cease, incomeSourceType), journeyState = InitialPage) { sessionData =>
+    withSessionData(JourneyType(Cease, incomeSourceType), journeyState = BeforeSubmissionPage) { sessionData =>
```

Comment:
After talking to Daniel, we agreed this also should not be an InitialPage, so changed to BeforeSubmissionPage.